app/controllers/school/details_controller.rb

```diff
@@ -0,0 +1,20 @@
+class School::DetailsController < School::BaseController
+  def show
+    @school_summary = details_rows
+  end
+
+private
+
+  def details_rows
+    [
+      {
+        key: 'Provisional allocation',
+        value: t(:n_devices, scope: %w[school details], count: @school.std_device_allocation&.allocation.to_i),
+      },
+      {
+        key: 'Will your school need to order Chromebooks?',
+        value: @school.preorder_information&.will_need_chromebooks.to_s.capitalize,
```

Comment:
```suggestion
        value: @school.preorder_information&.will_need_chromebooks&.to_s&.capitalize,
```
If you're using the `&` operator, you need to apply it to all subsequent chained calls otherwise it will blow up on the next call.

Reply:
....but `.to_s` converts nil to empty_string, meaning they're not needed

Reply:
In which case I'd add a `|| ""` clause to make the fall-through default really explicit...

Reply:
Isn't that just redundant? 

`nil.to_s == ""` has been in Ruby longer than I have (<2006), I'd argue that it's both more expected and more appropriate here than `&.` (which returns a nil you'd just have to handle elsewhere) and more elegant than `(some.chain.of.methods || "").some_other_method`

Reply:
I'm not going to die in a ditch over it, it's fine to leave as is, I just found it surprising/harder to parse than expected and would find the default more explicit hence flagged it here, but it's a matter of personal preference...